tests/test_entry_points.py

```diff
@@ -0,0 +1,32 @@
+from os.path import exists as path_exists
+
+from pyscaffold.api import create_project
+from pyscaffold.cli import parse_args
+from pyscaffold.contrib.configupdater import ConfigUpdater
+
+def test_entry_point_with_namespace(tmpfolder):
+    args = ["--namespace", "test", "--custom-extension", "some_extension"]
+
+    opts = parse_args(args)
+    create_project(opts)
+    assert path_exists("some_extension/setup.cfg")
+
+    config_updater = ConfigUpdater()
+    config_updater.read_file(open("some_extension/setup.cfg"))
```

Comment:
It's only a unit test but still I would recommend to use a context manager here for opening the file.